.devcontainers/OWNERS

```diff
@@ -0,0 +1,17 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+options:
+  # make root approval non-recursive
+  no_parent_owners: true
```

Comment:
Why is this necessary?

Reply:
That was my suggestion to mirror the .github directory that contribex maintains. We could potentially remove it as the root approvers are also well positioned to look at and test the changes

Reply:
yes pleqase

Reply:
+1